habitat-baselines/habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -148,11 +148,13 @@ def __init__(
     ):
         super().__init__()
         self.no_downscaling = no_downscaling
+        # Goal/privileged information sensors
```

Comment:
should we add an explanation?
also it seems weird that this is hard-coded?

Reply:
Added an explanation. @JHurricane96 why was this necessary?

Reply:
Removed now. This is not needed.